README.md

```diff
@@ -1,28 +1,167 @@
+
+<!-- omit in toc -->
+### Anatomy of a SRF Pipeline
+
+![SRF Pipeline](docs/imgs/srf_pipeline.jpg)
+
+<!-- omit in toc -->
+## Table of Contents
+- [Installation](#installation)
+  - [Prerequisites](#prerequisites)
+  - [Conda Installation](#conda-installation)
+    - [Python Bindings](#python-bindings)
+    - [C++ Bindings](#c-bindings)
+    - [Full SRF Library](#full-srf-library)
+  - [Source Installation](#source-installation)
+  - [Docker Installation](#docker-installation)
+- [Quickstart Guide](#quickstart-guide)
+- [Contributing](#contributing)
+  - [Third-Party code](#third-party-code)
+
+
+## Installation
+SRF includes both Python and C++ bindings and supports installation via [conda](https://docs.conda.io/en/latest/), Docker, or from source.]
+
+### Prerequisites
+
+- Pascal architecture (Compute capability 6.0) or better
+- NVIDIA driver `450.80.02` or higher
+- [conda or miniconda](https://conda.io/projects/conda/en/latest/user-guide/install/linux.html)
+- If using Docker:
+  - [Docker](https://docs.docker.com/get-docker/)
+  - [The NVIDIA container toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#docker)
+
+### Conda Installation
+Installing via Conda is the easiest method for getting the SRF components and supports both the Python and C++ bindings. To install the SRF conda package and build the C++ and hybrid components, follow the steps below. Please note, Conda performance can be greatly increased via [Mamba](https://github.com/mamba-org/mamba). To install Mamba in an existing Conda environment, simply run:
+
+```bash
+conda install -c conda-forge mamba
+```
+
+If you choose to use Mamba rather than Conda, simply replace `conda` with `mamba` in the instructions below.
+
+To avoid specifying the channels in the Conda commands below:
+```bash
+conda config --env --add channels conda-forge &&\
+conda config --env --add channels nvidia &&\
+conda config --env --add channels rapidsai
+```
+
+#### Python Bindings
+```bash
+# If needed, create a new conda environment
+conda create --name srf python=3.8
+
+# Activate the newly created conda environment
+conda activate srf
+
+# Install SRF Python bindings
+conda install -c rapidsai -c nvidia -c conda-forge srf
+```
 
-## Quickstart
+#### C++ Bindings
+```bash
+# If needed, create a new conda environment
+conda create --name srf python=3.8
 
-To get started with SRF, see the SRF Quickstart Repository located [here](/docs/quickstart/README.md).
+# Activate the newly created conda environment
+conda activate srf
+
+# Install SRF Python bindings
+conda install -c rapidsai -c nvidia -c conda-forge libsrf
+```
+
+#### Full SRF Library
+```bash
+# If needed, create a new conda environment
+conda create --name srf python=3.8
+
+# Activate the newly created conda environment
+conda activate srf
+
+# Install SRF Python bindings
+conda install -c rapidsai -c nvidia -c conda-forge srf libsrf
+```
+
+### Source Installation
```

Comment:
is this redundant with contributing?  idk

Reply:
I personally wouldn't look in a contributing guide to find instructions to build from source. Let's put this info up front and easy to find.